core/src/main/scala/org/bitcoins/core/protocol/ln/LnInvoiceTags.scala

```diff
@@ -138,7 +207,7 @@ sealed abstract class LnInvoiceTags {
     val quotient = Bech32EncodedString.length / 32
     val remainder = Bech32EncodedString.length % 32
     val v = Vector(UInt8(quotient.toShort), UInt8(remainder.toShort))
-    Bech32.encode8bitToString(v)
+    Bech32.encode5bitToString(v)
```

Comment:
This function is orphaned, and should be removed. Functionality has been moved to `dataLength` and `encodeNumber`.